pom.xml

```diff
@@ -154,7 +154,7 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
-		</dependencies>
+        </dependencies>
```

Comment:
Spaces are used and also it's an artifact left over from your previous dependency addition.